src/components/tagsInput/index.js

```diff
@@ -24,7 +24,6 @@ const GUTTER_SPACING = 8;
  * @gif: https://camo.githubusercontent.com/9c2671024f60566b980638ea01b517f6fb509d0b/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f336f45686e374a79685431566658746963452f67697068792e676966
  * @example: https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ChipsInputScreen.js
  * @extends: TextField
- * @extendsLink: https://github.com/wix/react-native-ui-lib/blob/master/src/components/inputs/TextField.js
```

Comment:
I see you've removed some links instead of replacing them, why?

Reply:
Cause our docs internally knows to refer to other components by their name. 
In this case, TagsInput is based on our TextField component, so the name is enough. 
The TextField component will have an external link (`@extendsLink`) that directs to RN's TextInput component